managers/docker-manager/DockerService.js

```diff
@@ -0,0 +1,801 @@
+        .filter(volume => _.startsWith(volume.Name, this.containerName))
+        .value()
+      )
+      .map(volume => this.removeVolume(volume.Name))
+      .return();
+  }
+
+  getNetworkInfo(containerInfo) {
+    const info = {
+      ports: {}
+    };
+    _.each(containerInfo.NetworkSettings.Ports, (values, key) => {
+      if (_.size(values)) {
+        const value = _.first(values);
+        info.ip = value.HostIp;
+        info.ports[key] = value.HostPort;
+      }
+    });
+    if (info.ip === '0.0.0.0') {
+      info.ip = this.hostIp;
+    }
+    return info;
+  }
+
+  // getProcesses() {
+  //   if (this.containerInfo.State.Running) {
+  //     return this.container
+  //       .topAsync({
+  //         ps_args: 'aux'
+  //       })
+  //       .then(top => _.concat([top.Titles], top.Processes));
+  //   }
+  // }
+
+  // getLogs() {
+  //   return this.container
+  //     .logsAsync({
+  //       stdout: 1,
+  //       stderr: 1,
+  //       timestamps: 1
+  //     })
+  //     .then(stream => utils.demux(stream, {
+  //       tail: 1000
+  //     }));
+  // }
+
+  // getInfo() {
+  //   return Promise
+  //     .all([
+  //       this.cloudController.getServiceInstance(this.guid),
+  //       this.inspectContainer()
+  //     ])
+  //     .spread((instance, containerInfo) => {
+  //       /* jshint unused:false */
+  //       return Promise.all([
+  //         instance,
+  //         this.getDetails(),
+  //         this.getProcesses(),
+  //         this.getLogs(),
+  //         this.cloudController.getServicePlan(instance.entity.service_plan_guid, {})
+  //       ]);
+  //     })
+  //     .spread((instance, details, processes, logs, planInfo) => {
+  //       var currentPlan = catalog.getPlan(planInfo.entity.unique_id);
+  //       return {
+  //         title: `${currentPlan.service.metadata.displayName || 'Service'} Dashboard`,
+  //         plan: currentPlan,
+  //         service: currentPlan.service,
+  //         instance: instance,
+  //         details: details,
+  //         processes: processes,
+  //         files: [{
+  //           id: 'stdout',
+  //           title: 'Standard',
+  //           language: 'ansi',
+  //           content: logs[0]
+  //         }, {
+  //           id: 'stderr',
+  //           title: 'Error',
+  //           language: 'xxxx',
+  //           content: logs[1]
+  //         }]
+  //       };
+  //     });
+  // }
+
+  // getContainerStatus(state) {
+  //   if (state.Running) {
+  //     return `Up for ${utils.getTimeAgo(state.StartedAt, true)}${state.Paused ? ' (Paused)' : ''}`;
+  //   }
+  //   if (state.ExitCode > 0) {
+  //     return `Exited (${state.ExitCode}) ${utils.getTimeAgo(state.FinishedAt)}`;
+  //   }
+  //   return 'Stopped';
+  // }
+
+  getEnvironment() {
+    return _
+      .chain(this.containerInfo)
+      .get('Config.Env')
+      .map(kv => _.slice(/^([^=]+)=(.*)$/.exec(kv), 1))
+      .fromPairs()
+      .value();
+  }
+
+  // getDetails() {
```

Comment:
no commented out code in code base.

Reply:
This methods are copied from old Instance/Manager files. They are still used with the old flow as part of dashboard show, which should eventually be removed. Kept it here so that we can uncomment it here and remove from old Instance/Manager files. Also we decided not to touch the Instance/Manager files in the current PR. They also will be cleaned up.